docs/installation/mac.md

```diff
@@ -107,16 +107,26 @@ and choosing "Open" from the pop-up menu.
 
 7. Press "Close" to exit.
 
+## Learn about your Toolbox installation
```

Comment:
Moved to old position. This section needs a rewrite and restructure-will look at it.